packages/inspect/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Injects an inspector into an Effection application",
   "main": "index.cjs.js",
   "module": "index.esm.js",
+  "sideEffects": false,
```

Comment:
If anything has side effects, it is this one. This package is meant to be used specifically for its side-effects:

e.g.

```
node -r @effection/inspect my-script.js
```

Reply:
removed